cluster/service/cluster_service.go

```diff
@@ -147,21 +161,15 @@ func validate(clustr *repository.Cluster) error {
 
 // validateURL validates the URL: return `false` if the given url could not be parsed or if it is missing
```

Comment:
Need to fix the comment. It returns error now.

Reply:
fixed in 790c3b0